docs/building-spark.md

```diff
@@ -161,6 +161,8 @@ For help in setting up IntelliJ IDEA or Eclipse for Spark development, and troub
 
 # Building Spark Debian Packages
 
+_NOTE: Debian packaging is deprecated and will be removed in a future release._
```

Comment:
Should we maybe just say "in Spark 1.4"? It might be good to set expectations conservatively... no harm if we decide to keep it longer.